debian/control.in

```diff
@@ -0,0 +1,14 @@
+Source: pg-bitcount
+Maintainer: Gijs Kant <gijs@thehyve.nl>
+Section: database
+Priority: optional
+Standards-Version: 4.2.1
+Build-Depends: postgresql-server-dev-PGVERSION
+
+Package: postgresql-PGVERSION-pg-bitcount
+Architecture: any
```

Comment:
Since for other architectures we need another pipeline. And `any` is applicable for packages that are realy architecture agnostic, like bash-scripts, python-scripts and etc.

Reply:
Actually `all` means architecture independent, see https://www.debian.org/doc/debian-policy/ch-controlfields.html#architecture. `any` means that it should compile fine on any architecture.